src/controller/main.h

```diff
@@ -12,15 +12,37 @@
 
 //#define DEBUG_UART
 
-#define PWM_CYCLES_COUNTER_MAX                    3125    // 5 erps minimum speed; 1/5 = 200ms; 200ms/64us = 3125
-#define PWM_CYCLES_SECOND                         15625L  // 1 / 64us(PWM period)
-#define PWM_DUTY_CYCLE_MAX                        254
-#define PWM_DUTY_CYCLE_MIN                        20
-#define MIDDLE_PWM_DUTY_CYCLE_MAX                 (PWM_DUTY_CYCLE_MAX/2)
 
 
-// motor rotor offset 
-#define MOTOR_ROTOR_OFFSET_ANGLE                  10
+// motor 
+#define PWM_CYCLES_COUNTER_MAX                                    3125    // 5 erps minimum speed; 1/5 = 200ms; 200ms/64us = 3125
+#define PWM_CYCLES_SECOND                                         15625L  // 1 / 64us(PWM period)
+#define PWM_DUTY_CYCLE_MAX                                        254
+#define PWM_DUTY_CYCLE_MIN                                        0
+#define MIDDLE_PWM_DUTY_CYCLE_MAX                                 127     // PWM_DUTY_CYCLE_MAX / 2
+
+#define PWM_DUTY_CYCLE_RAMP_UP_INVERSE_STEP                       60      // 60 -> 60 * 64 us for every duty cycle increment
```

Comment:
Why did you change this from 20 to a higher value ?

Reply:
WIP.